### ISSUE TITLE
Unmap key <20> (Volume control)

### ISSUE THREAD

**Issue by mangelozzi** [neovim/neovim#12756]
- `nvim --version`:
```
:ver
NVIM v0.5.0-563-ge628a05b5
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/gcc-5 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict
-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto
-DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/travis/b
uild/neovim/bot-ci/build/neovim/build/config -I/home/travis/build/neovim/bot-ci/build/neovim/src -I/home/travis/build/neovim/bot-ci/bu
ild/neovim/.deps/usr/include -I/usr/include -I/home/travis/build/neovim/bot-ci/build/neovim/build/src/nvim/auto -I/home/travis/build/n
eovim/bot-ci/build/neovim/build/include
Compiled by travis@travis-job-c5758e75-4445-4564-8946-810c98499a51
```
- Operating system/version: Windows 10 running WSL2 Ubuntu 20.04
- `$TERM`: xterm-256color

### Steps to reproduce using `nvim -u NORC`

```
nvim -u NORC
# Alternative for shell-related problems:
# env -i TERM=ansi-256color "$(which nvim)"
```
1. Have a keyboard with volume up / down keys.
2. Type a few words, place cursor in the middle of the buffer.
3. Press Volume up or voume down key.

### Actual behaviour
In normal mode the cursor jumps forward a WORD at a time (as if pressing `W`), happens with both volume up or down. In insert mode it seems to paste from  `".` then escape back to normal mode.

Tried with Vim and got the same result.

### Expected behaviour
Nothing to happen.

### Further info
When I press the key, bottom right where Neovim shows which key is pressed, it shows `<20>` as the key being pressed:
![image](https://user-images.githubusercontent.com/19764314/90315146-5e731c00-df19-11ea-8057-7c75cb70805f.png)

I see char 20 (decimal) is `device-control 4` (<http://www.asciitable.com/>)
So I tried (after some research):
```vim
:unmap <20>
:unmap <Char-20>
:unmap <Char-0x20>
```
But I get:
```vim
E31 No such mapping
```

**Comment by justinmk** [comment]
It's not mapped, so unmapping it does nothing. Your terminal is sending this sequence and then vim doesn't know what to do with it, so it just inserts (some part of?) the bytes. To see the full sequence sent by the terminal, try this in a shell (not vim):

    $ cat -v

Then you can try 

    :noremap! {sequence} <nop>
    :noremap {sequence} <nop>

Closing since this is not a bug and there's nothing we can/should do about it. But feel free to comment further, mentioning the full sequence that is reported by `cat -v`.

**Comment by mangelozzi** [comment]
Thanks Justin for looking at my humble request and giving a good lead ^_^!

So `cat -v` printed out `^@` for both volume up and down. Then trying to map `<C-@>` to `<nop>`, it did not prevent volumes keys from performing. Mapping `<Char-20` to `<nop>` also didn't work.

For anyone else who has this issue:
In insert mode type CTRL+v then press volume up. It will insert the sequence. Unfortunately it only prints the first char, which my case was `<C-Space>`. 
There a string of other chars afterwards:
![image](https://user-images.githubusercontent.com/19764314/90492270-4c73c200-e141-11ea-9de5-21fbea7e3feb.png)

Which I was unsuccessful in mapping. But just mapping the `<C-Space>` was enough to disable the volume keys from performing actions.

In summary add this:
```vim
" Keyboard volume up and down and down to no operation
noremap! <C-Space> <nop>
noremap  <C-Space> <nop>
```
